Tasks/Common/kubernetes-common-v2/image-metadata-helper.ts

```diff
@@ -192,19 +195,128 @@ function getServerUrl(clusterInfo: any): string {
     return serverUrl;
 }
 
-function getManifestFilePaths(): string[] {
-    let manifestFilePaths: string[] = [];
-    const commandArguments = tl.getInput("arguments", false);
-    const filePathMatch: string[] = commandArguments.split(matchPatternForFileArgument);
+export function extractManifestsFromHelmOutput(helmOutput: string): any {
+    let manifestObjects = [];
+    let manifestFiles = "";
+    // The output stream contains the manifest file between the manifest and last deployed fields
+    const manifestString = "manifest:";
+    const lastDeployedString = "last deployed:";
+    let indexOfManifests = helmOutput.toLowerCase().indexOf(manifestString);
+    let indexOfLastDeployed = helmOutput.toLowerCase().indexOf(lastDeployedString);
+    if (indexOfManifests >= 0 && indexOfLastDeployed >= 0) {
+        manifestFiles = helmOutput.substring(indexOfManifests + manifestString.length, indexOfLastDeployed);
+    }
+
+    if (manifestFiles) {
+        // Each of the source manifests is separated in output stream via string '---'
+        const files = manifestFiles.split("---");
+        files.forEach(file => {
+            file = file.trim();
+            if (file) {
+                const parsedObject = yaml.safeLoad(file);
+                manifestObjects.push(parsedObject);
+            }
+        });
+    }
+
+    return manifestObjects;
+}
+
+export function getManifestFileUrlsFromArgumentsInput(fileArgs: string): string[] {
+    let manifestFileUrls: string[] = [];
+    const filePathMatch: string[] = fileArgs.split(matchPatternForFileArgument);
```

Comment:
I have updated the code so that we either send inline file path if provided, else check for arguments which will not be null if the test reaches this piece of code.
Anyways sending fileArgs as an empty string here, so fileArgs.split will not throw null exception